lib/households/view/households.dart

```diff
@@ -0,0 +1,128 @@
+          HouseholdsBloc(isarRepository: context.read<IsarRepository>())
+            ..add(const HouseholdsPageOpened()),
+      child: const HouseholdsView(),
+    );
+  }
+}
+
+class HouseholdsView extends StatelessWidget {
+  const HouseholdsView({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    DateFormat formatter = DateFormat('yyyy-MM-dd');
+
+    return BlocBuilder<HouseholdsBloc, HouseholdsState>(
+      builder: (context, state) {
+        return Scaffold(
+            appBar: AppBar(
+              title: const Text('Households'),
+              actions: [
+                IconButton(
+                    onPressed: () => Navigator.pushNamed(
+                        context, PageRouter.collectionsHelp),
+                    icon: const Icon(Icons.help))
+              ],
+            ),
+            body: Column(
+              children: [
+                Expanded(
+                  child: ListView.builder(
+                    padding: const EdgeInsets.all(2.0),
+                    itemCount: state.households.length,
+                    itemBuilder: (context, index) {
+                      return Card(
+                          child: ListTile(
+                        title: Text(state.households[index].householdId),
+                        subtitle: Text(formatter
+                            .format(state.households[index].sensitizationDate)),
+                        onTap: () => Navigator.pushNamed(
+                            context,
+                            PageRouter.householdPrefix +
+                                PageRouter.viewHousehold,
+                            arguments: {'id': state.households[index].id}),
+                        onLongPress: () => showModalBottomSheet(
+                            context: context,
+                            builder: (context) {
+                              return HouseholdOptions(
+                                  id: state.households[index].id,
+                                  householdId:
+                                      state.households[index].householdId);
+                            }),
+                      ));
+                    },
+                  ),
+                )
+              ],
+            ),
+            floatingActionButton: Column(
+                mainAxisAlignment: MainAxisAlignment.end,
+                crossAxisAlignment: CrossAxisAlignment.end,
+                children: <Widget>[
+                  FloatingActionButton.extended(
+                      heroTag: null,
+                      label: const Text("New household"),
+                      icon: const Icon(Icons.add),
+                      onPressed: () {
+                        Navigator.pushNamed(
+                            context, PageRouter.createHousehold);
+                      }),
+                ]));
+      },
+    );
+  }
+}
+
+class HouseholdOptions extends StatelessWidget {
```

Comment:
here too

Reply:
Since this is constructing a bottom sheet rather than a widget inside the existing scaffold I'd prefer to keep it separate